index.js

```diff
@@ -211,28 +203,28 @@ class ExecutorQueue extends Executor {
      * @return Promise.resolve
      */
     async api(config, args) {
+        const { token } = config;
+
+        delete config.token;
+
         const options = {
             headers: {
-                Authorization: `Bearer ${config.token}`,
+                Authorization: `Bearer ${token}`,
                 'Content-Type': 'application/json'
             },
             url: `${this.queueUri}${args.path}`,
             json: true,
             maxAttempts: RETRY_LIMIT,
             retryDelay: RETRY_DELAY * 1000, // in ms
             retryStrategy: this.requestRetryStrategy,
-            ...args
+            ...args,
+            body: config
         };
 
         return new Promise((resolve, reject) => {
             requestretry(options, (err, response) => {
                 if (!err) {
-                    if (response.statusCode === 200) {
-                        return resolve(response);
```

Comment:
what changed ? don't we have this case anymore ?

Reply:
We are returning everything, irrespective of statusCode